pkg/v1/remote/schema1.go

```diff
@@ -71,7 +71,18 @@ func (s *schema1) ConfigName() (v1.Hash, error) {
 }
 
 func (s *schema1) ConfigFile() (*v1.ConfigFile, error) {
-	return nil, newErrSchema1(s.mediaType)
+	configFile := &v1.ConfigFile{
+		RootFS: v1.RootFS{},
+	}
+	layers, err := s.Layers()
+	if err != nil {
+		return nil, err
+	}
+	for _, layer := range layers {
+		d, _ := layer.Digest()
```

Comment:
can layers be compressed in this case?

Reply:
yes layers are compressed, except for the manifest - but it doesn't change the handling